docs/src/pages/demos/progress/DelayingAppearance.js

```diff
@@ -23,14 +23,7 @@ function DelayingAppearance() {
   const classes = useStyles();
   const [loading, setLoading] = React.useState(false);
   const [query, setQuery] = React.useState('idle');
-  let timer = null;
-
-  React.useState(
-    () => () => {
-      clearTimeout(timer);
```

Comment:
Shouldn't we clear the timer when the component unmount?

Reply:
@oliviertassinari Fixed! Sorry about that!